tests/unit/utils/test_config.py

```diff
@@ -167,101 +166,6 @@ def test_invalid_config():
         "provider URL is invalid",
     )
 
-    for role in constants.PROVIDER_MODEL_ROLES:
-        check_expected_exception(
-            f"""
----
-llm_providers:
-  - name: p1
-    type: bam
-    url: 'http://url1'
-    models:
-      - name: m1
-        url: 'http://murl1'
-dev_config:
-  disable_tls: true
-ols_config:
-  conversation_cache:
-    type: memory
-    memory:
-      max_entries: 1000
-  {role}_provider: no_such_provider
-  {role}_model: m1
-    """,
-            InvalidConfigurationError,
-            f"{role}_provider specifies an unknown provider no_such_provider",
-        )
-
-        check_expected_exception(
-            f"""
----
-llm_providers:
-  - name: p1
-    type: bam
-    url: 'http://url1'
-    models:
-      - name: m1
-        url: 'http://murl1'
-dev_config:
-  disable_tls: true
-ols_config:
-  conversation_cache:
-    type: memory
-    memory:
-      max_entries: 1000
-  {role}_provider: p1
-  {role}_model: no_such_model
-    """,
-            InvalidConfigurationError,
-            f"{role}_model specifies an unknown model no_such_model",
-        )
-
-        check_expected_exception(
-            f"""
----
-llm_providers:
-  - name: p1
-    type: bam
-    url: 'http://url1'
-    models:
-      - name: m1
-        url: 'http://murl1'
-dev_config:
-  disable_tls: true
-ols_config:
-  conversation_cache:
-    type: memory
-    memory:
-      max_entries: 1000
-  {role}_provider: p1
-    """,
-            InvalidConfigurationError,
-            f"{role}_provider is specified, but {role}_model is missing",
-        )
-
-        check_expected_exception(
-            f"""
----
-llm_providers:
-  - name: p1
-    type: bam
-    url: 'http://url1'
-    models:
-      - name: m1
-        url: 'http://murl1'
-dev_config:
-  disable_tls: true
-ols_config:
-  conversation_cache:
-    type: memory
-    memory:
-      max_entries: 1000
-  {role}_model: m1
-    """,
-            InvalidConfigurationError,
-            f"{role}_model is specified, but {role}_provider is missing",
-        )
-
     check_expected_exception(
```

Comment:
I didn't mean to remove these tests completely, just remove the loop and hadcode the role parameter to "default"

Reply:
ah sure thing, sorry for misunderstanding. updated